plugin/hover.py

```diff
@@ -83,7 +83,13 @@ def link(self, point: int, view: sublime.View) -> str:
 ]
 
 
-def code_actions_content(actions_by_config: list[CodeActionsByConfigName]) -> str:
+LIGHTBULB_ICON_HTML = """
+<span class="lightbulb">
+    <img width="13" height="13" src="res://Packages/LSP/icons/lightbulb_colored.png">
+</span>"""
```

Comment:
- we don't really need to wrap it in `span`
- Use em units so that the icon size better adapts to whatever font size the user is using

```suggestion
    <img src="res://Packages/LSP/icons/lightbulb_colored.png" style="width: 1em; height: 1em">
"""
```

(just an example, you can of course move the styles to css)

Reply:
I tried to set the style (padding-right) directly on the img element, but then it seems to be ignored. That's probably a limitation/bug in ST minihtml. That was the reason for the span wrapper.

I will update the units.

Reply:
Try margin instead

Reply:
Tried that as well, but without success.

Reply:
Since it's an inline (or inline-block) element it should be then enough to create an extra spacing by having a space characters between the icon and the text

Reply:
Hm, not a big fan of using space characters or `&nbsp` as padding/spacers. I hate when websites do this and you accidentally select them when you want to select some text. And maybe one day ST might make the contents of popups selectable. The CSS styling on the span is also a more flexible solution.